### PR TITLE
Nidoking/eng 2860 clean webpack and merge tooltip pr feedback

### PR DESCRIPTION
**Pull request by kafkasamsa**

### CODE REVIEW

File: src/components/Tooltip/Components.js [hunk 1]
```diff
@@ -102,8 +109,26 @@ class InfoBox extends React.Component {
         }
     }
 
+    disableOutOfViewportCheck(device) {
+        const { position } = this.props;
+        const node = this.refs.element;
+        const isMobile = Utils.checkForDevices('mobile');
+        const isTablet = Utils.checkForDevices('tablet');
+
+        if (
```
[the remark this answers is not in the document]
Comment (cawo88) [reply src/components/Tooltip/Components.js]: This `if` statement is used to check which device(s) (props) the user want to disable out of viewport detection and passing the `device` argument to the `Utils.checkForDevices` method is not the desire outcome.

File: src/components/Tooltip/Components.js [hunk 2]
```diff
@@ -350,19 +381,23 @@ class InfoBox extends React.Component {
 
     arrowPosition(pos) {
         const { top, right, bottom, left, centerX, centerY } = this.props;
-        const arrowVerticalWidth = 4; // TODO: could dynamically generate this value
+        const arrowVerticalWidth = 4;
```
[the remark this answers is not in the document]
Comment (cawo88) [reply src/components/Tooltip/Components.js]: No unfortunately, I would like to but it's a bit tricky so will leave it as it in :(

File: package.json [hunk 3]
```diff
@@ -4,13 +4,12 @@
     "description": "React components library",
     "main": "build/index.js",
     "scripts": {
-        "storybook": "start-storybook -p 6006",
-        "prettier": "prettier --config \".prettierrc\" --write \"**/*.{js,jsx,json,scss,css,ts}\"",
-        "start": "node scripts/start.js",
+        "storybook": "start-storybook -p 9001",
```
[the remark this answers is not in the document]
Comment (cawo88) [reply package.json]: Could you have a look at this please @kafkasamsa when you're free. I went through the feedback for tooltip already. Thanks in advance.

[the remark this answers is not in the document]
Comment (kafkasamsa) [reply package.json]: The start was something I added and removed. So we use the storybook for running locally and build for building the index file.

**Comment by onlyahost** [comment]
@radibit @cawo88 
From a code perspective everything looks fine for me.
Has anyone tested this PR with using this specific branch directly in the app / website?
If someone did and everything worked, I think we are good to go.

**Comment by radibit** [comment]
@onlyahost @cawo88 @dcastil 

I didn't test that yet. It'll be nice if someone who is familiar with the tooltip requirements do this (I'm not able to find a ticket on Jira). What do you think?

**Comment by cawo88** [comment]
I tested the tooltip using this specific branch and it's fine. 

If anything, maybe @dcastil wouldn't mind doing a QA for the other components like the modal, select menu, heading, and grid ( if we are using it in the web-app)? I will move this ticket into the 'In QA' column in JIRA @kafkasamsa, hope you don't mind me doing that.

**Pull request merged**